### ISSUE TITLE
Hard to notice difference in test results with changed json file

### ISSUE THREAD

**Issue by artokin** [ARMmbed/mbed-os-example-nvstore#4]
In Step 9 "Repeat steps 3-7. Notice the changes in prints of area addresses and sizes and with the amount of possible keys this configuration can hold."

It is hard to notice difference in prints when using `mbed_app.json` vs. `mbed_app-8KB-areas.json`:

mbed_app.json
```
--- Mbed OS NVStore example ---
Init NVStore. Return code is 0
NVStore size is 4096
NVStore max number of keys is 16 (out of 255 possible ones in this flash configuration)
NVStore areas:
Area 0: address 0x0007e000, size 4096 (0x1000)
Area 1: address 0x0007f000, size 4096 (0x1000)
Reset NVStore. Return code is 0
Set key 1 to value 1000. Return code is 0
Set key 1 to value 2000. Return code is 0
Set key 1 to value 3000. Return code is 0
Get key 1. Value is 3000. Return code is 0
Delete key 1. Return code is 0
Get key 1. Return code is -3
Set key 12 once to value 50. Return code is 0
Set key 12 to value 100. Return code is -9
Get key 12. Value is 50. Return code is 0
Data size for key 12 is 4. Return code is 0

--- Mbed OS NVStore example done. ---
```

mbed_app-8KB-areas.json
```
--- Mbed OS NVStore example ---
Init NVStore. Return code is 0
NVStore size is 4096
NVStore max number of keys is 16 (out of 255 possible ones in this flash configuration)
NVStore areas:
Area 0: address 0x0007e000, size 4096 (0x1000)
Area 1: address 0x0007f000, size 4096 (0x1000)
Reset NVStore. Return code is 0
Set key 1 to value 1000. Return code is 0
Set key 1 to value 2000. Return code is 0
Set key 1 to value 3000. Return code is 0
Get key 1. Value is 3000. Return code is 0
Delete key 1. Return code is 0
Get key 1. Return code is -3
Set key 12 once to value 50. Return code is 0
Set key 12 to value 100. Return code is -9
Get key 12. Value is 50. Return code is 0
Data size for key 12 is 4. Return code is 0

--- Mbed OS NVStore example done. ---
```

**Comment by davidsaada** [comment]
@artokin Which board did you test this example on? The changes caused by the switch of JSON files are noticeable only on K64F. As these changes deal with different sector sizes (which vary from one board to another), the 8KB areas JSON file won't fit other boards.
I will rename this file so it will be clear that it only fits K64F, and fix the text in README.md, so it's clear that step 9 is only relevant for this board.

**Comment by artokin** [comment]
Thanks @davidsaada , Next time I'll try with K64F as well.

**Comment by adbridge** [comment]
@davidsaada can you advise the PR you will fix the readme on and link here please. It needs to go in for RC2 on Monday. Thanks

**Comment by davidsaada** [comment]
@adbridge Pushed a fix for it now.

**Comment by adbridge** [comment]
@davidsaada Can you reference the Pull Request here please and close this if now fixed

**Comment by davidsaada** [comment]
Fixed in commit `8f0fcb2`
